script/gitlabci/job_cuda11.6.yml

```diff
@@ -113,3 +113,28 @@ linux_nvcc-11.6_clang-13_release:
     CMAKE_CUDA_COMPILER: nvcc
     alpaka_ACC_CPU_B_OMP2_T_SEQ_ENABLE: "OFF" # bug, see: https://github.com/NVIDIA/cuda-samples/issues/102
   stage: stageRun1
+
+# nvhpc
+linux_nvhpc-22.9_cuda-11.6_oacc:
+  extends: .base_oacc_nvhpc
+  variables:
+    ALPAKA_CI_UBUNTU_VER: "20.04"
+    ALPAKA_CI_CUDA_VERSION: "11.6"
```

Comment:
It looks like you need to change this to 11.7 like for the github action job. Current error message is: 

```
ERROR: Job failed: failed to pull image "nvcr.io/nvidia/nvhpc:22.9-devel-cuda11.6-ubuntu20.04" with specified policies [always]: Error response from daemon: manifest for nvcr.io/nvidia/nvhpc:22.9-devel-cuda11.6-ubuntu20.04 not found: manifest unknown: manifest unknown (manager.go:235:2s)
```